projects/aca-content/src/lib/components/libraries/libraries.component.scss

```diff
@@ -0,0 +1,52 @@
+.aca-search-input {
```

Comment:
copy-pasted code across multiple files, please move out to a shared scss file, somewhere in the `projects/aca-content/src/lib/ui`